cmd/kpod/common.go

```diff
@@ -31,3 +32,30 @@ func getStore(c *cli.Context) (storage.Store, error) {
 	is.Transport.SetStore(store)
 	return store, nil
 }
+
+func getConfig(c *cli.Context) (*libkpod.Config, error) {
+	config := libkpod.DefaultConfig()
+	if c.GlobalIsSet("config") {
+		err := config.FromFile(c.String("config"))
+		if err != nil {
+			return config, err
+		}
+	}
+	if c.GlobalIsSet("root") {
```

Comment:
There's a set of global flags in main.go and every subcommand should have access to it, but why do we need to append the flags to each command if they can already be accessed globally?